pytorch_lightning/trainer/training_loop.py

```diff
@@ -876,11 +877,29 @@ def build_train_args(self, batch, batch_idx, opt_idx, hiddens):
                 )
 
         # pass hiddens if using tbptt
-        if self.trainer.truncated_bptt_steps is not None:
+        if self._truncated_bptt_enabled():
             args.append(hiddens)
 
         return args
 
+    def _truncated_bptt_enabled(self) -> bool:
+        """ Temporary tbptt utilities until this flag is fully migrated to the lightning module. """
+        module = self.trainer.lightning_module
+        module_tbptt_enabled = module.truncated_bptt_steps > 0
+        if module_tbptt_enabled:
+            return True
+
+        trainer = self.trainer
+        trainer_tbptt_enabled = trainer.truncated_bptt_steps is not None and trainer.truncated_bptt_steps > 0
+        return trainer_tbptt_enabled
+
+    def _truncated_bptt_steps(self) -> Optional[int]:
+        lightning_module = self.trainer.lightning_module
+        # Give precedence to the LightningModule as the Trainer flag will be removed in v1.5
+        if lightning_module.truncated_bptt_steps > 0:
+            return lightning_module.truncated_bptt_steps
+        return self.trainer.truncated_bptt_steps
```

Comment:
```suggestion
        return self.trainer.truncated_bptt_steps or 0
```

This way we can remove the Optional annotation